### DISCUSSION TITLE
2nd level for subfolders

### DISCUSSION THREAD

**Discussion by PapiFed** [OnionUI/Onion#1001] [Ideas]
Hi Team,

Is there any plan to implement the ability to have more than 1 level of subfolder within the roms folder of a particular system ?
I would not consider this as a must have but it would help carrying over currated roms collection with a more advanced folder tree.

Big thanks for bringing the OS as far as you did already, it's really great !

**Comment by Aemiii91** [comment]
This is bound by Miyoo's closed source frontend that we're using. Implementing this would mean we need to write an entire new frontend.

**Reply by PapiFed** [reply]
Thank you for the prompt answer !
